src/information-panel/information-panel.scss

```diff
@@ -0,0 +1,145 @@
+
+@mixin iui-information-panel-wrapper {
+  position: relative;
+  overflow: hidden;
+}
+
+@mixin iui-information-panel {
+  position: absolute;
+  top: 0;
+  right: ($iui-3xl * 4) * -1; // Default width set negative so the panel hides off screen.
+  opacity: 0;
+  display: flex;
+  flex-direction: column;
+  width: $iui-3xl * 4;
+  min-width: $iui-3xl * 2;
+  height: 100%;
+  box-sizing: border-box;
+  box-shadow: -1px 0 14px rgba(0, 0, 0, 0.25);
+  transition: right $iui-speed-fast ease-out, opacity $iui-speed-fast ease;
+  z-index: 2; // Needs to be higher than table's column resizers.
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+
+  > .iui-resizer {
+    display: none;
+    height: 100%;
+    width: $iui-m;
+    position: absolute;
+    top: 0;
+    left: -17px; // Why?
+    transform: translateX(50%);
+    touch-action: none;
+    cursor: ew-resize;
+    z-index: 1000;
+
+    > .iui-resizer-bar {
+      height: 100%;
+      width: 1px;
+      margin: 0 auto;
+      @media (prefers-reduced-motion: no-preference) {
+        transition: background-color $iui-speed-fast ease-out, width $iui-speed-fast ease-out;
+      }
+      @include themed {
+        background-color: t(iui-color-background-5);
+      }
+    }
+
+    &:hover > .iui-resizer-bar {
+      width: $iui-xs;
+      @include themed {
+        background-color: t(iui-color-foreground-primary);
+      }
+    }
+  }
+
+  &.iui-information-panel-visible {
+    right: 0;
+    opacity: 1;
+
+    > .iui-resizer {
+      display: block;
+    }
+  }
+
+  .iui-information-header,
+  .iui-information-body {
+    padding-left: $iui-sm;
+    padding-right: $iui-sm;
+    box-sizing: border-box;
+  }
+
+  .iui-information-header {
+    width: 100%;
+    height: $iui-baseline * 6;
+    display: flex;
+    align-items: center;
+    @include themed {
+      background-color: t(iui-color-background-3);
+    }
+
+    .iui-selection-icon {
+      width: 24px;
+      height: 24px;
+      margin-right: 8px;
+    }
+
+    .iui-selection {
+      margin-right: auto;
+    }
+  }
+
+  .iui-information-body {
+    padding-top: $iui-baseline;
+    padding-bottom: $iui-baseline;
+    height: 100%;
+    overflow-x: hidden;
+    overflow-y: overlay;
+
+    > hr {
+      @include themed {
+        border-color: t(iui-color-background-4);
+      }
+    }
+  }
+
+  .iui-information-table {
+    table-layout: fixed;
+    width: 100%;
+
+    tr {
+      height: $iui-baseline * 3;
+
+      td {
+        vertical-align: middle;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+
+        &:first-child {
+          text-align: right;
+          color: t(iui-text-color-muted);
+          padding-right: $iui-s;
+
+          &::after {
+            content: ':';
+          }
+        }
+      }
+    }
+
+    .iui-information-table-center {
+      display: flex;
+      align-items: center;
+      gap: $iui-s;
+    }
+
+    label {
```

Comment:
expectation is to use our labeled input here, so we dont need to do anything with them..

Reply:
I tried this after reading your comment.  I changed the code from this…
```
<td>
  <label for="year">Year recorded</label>
</td>
<td>
  <input id="year" value="2021" class="iui-input iui-small" />
</td>
```
…to this…
```
<td>
  <label class="iui-input-container" for="year">
    <div class="iui-label">Year recorded</div>
  </label>
</td>
<td>
  <input id="year" value="2021" class="iui-input iui-small" />
</td>
```
…and this was the end result. Musical Genre is unchanged, only Year Recorded was changed so we can visually compare the two.
![Screen Shot 2021-09-14 at 3 27 20 PM](https://user-images.githubusercontent.com/849817/133321434-639ab43b-ff96-417d-a682-035481223731.png)